src/data/exp.ts

```diff
@@ -1,6 +1,13 @@
 import { CommonColor, ShadowColor } from "#enums/color";
 import { GrowthRate } from "#enums/growth-rates";
 
+/**
+ * These are the hardcoded total exp values required for levels 1 to 100 ordered as
+ * Erratic, fast, medium fast, medium slow, slow, fluctuating
+ *
+ * Note that the first 100 values are here for optimization but this optimization
+ * is moot if there are any smoothing or past 100
```

Comment:
```suggestion
 * A look up table containing the total amount of EXP required to reach each level (for the first `100` levels), ordered by growth rate:
 * Erratic, fast, medium fast, medium slow, slow, fluctuating
```
Btw would this be better as something like this?
```ts
const expLevels = {
  [GrowthRate.ERATTIC]: [0, 15, ...],
  // ...
};

Reply:
Ideally, yes, but because `GrowthRate` is an enum starting with 0 then the current access pattern is still readable `const levelExp = expLevels[growthRate][level - 1];`